response_operations_ui/templates/collection_exercise/ce-sample-ci-section.html

```diff
@@ -1,66 +1,88 @@
+                            "value": "Sample & Collection instruments (CI)",
+                            "thClasses": "ons-u-fs-r--b"
+                        },
+                        {
+                            "value": "",
+                            "thClasses": "ons-u-ta"
+                        },
+                        {
+                            "value": "Uploaded " + '<em>' + sample.ingestDateTime[:-8] + '</em>' + sample.ingestDateTime[-8:],
+                            "thClasses": "ons-u-fs-r",
+                            "numeric": "true"
+                        },
+                    ],
+                    
+                    "trs": [
+                        {
+                            "tds": [
+                                {
+                                    "value": "Total businesses",
+                                    "tdClasses": "ons-u-ta"
+                                },
+                                {
+                                    "value": sample.totalSampleUnits,
+                                    "name": "total-instruments",
+                                    "tdClasses": "ons-u-ta-right"
+                                },
+                                {
+                                    "value": ' ',
+                                    "name": "upload-sample-time",
+                                }
+                            ]
+                        },
+                        {
+                            "tds": [
+                                {
+                                    "value": "Collection instruments",
+                                    "tdClasses": "ons-u-ta"
+                                },
+                                {
+                                    "value": sample.expectedCollectionInstruments,
+                                    "name": "total-instruments",
+                                    "tdClasses": "ons-u-ta-right"
+                                },
+                                {
+                                    "value": ' ',
+                                    "name": "upload-sample-time",
+                                }
+                            ]
+                        }
+                    ]
+                })
+            }}
+        {% elif 'INIT' == sample.state %}
+            <div class="ons-u-mb-xl">
+                <div class="ons-u-fs-r">
+                    Loading&hellip; <span class="ons-u-ml-xs"><a href="/surveys/{{survey.shortName}}/{{ce.exerciseRef}}/view-sample-ci?show_msg=true">Refresh to see progress</a></span>
+                </div>
+            </div>
+        {% elif 'FAILED' == sample.state %}
+            <div class="ons-u-mb-xl">
+                {% call
+                    onsPanel({
+                        "type": "error",
+                        "classes": "ons-u-mb-l"
+                    })
+                %}
+                    {{ sample.notes }}
```

Comment:
I didn't go to the extent of finding the text for sample.notes, is it consumable for a respondent.

Reply:
It shows the user where and what the error is:
`Line 2753: This sample unit ref 49907527428 is duplicated in the file.`
`Line 1: Error in [11110000001, H, 75110.0, 75110.0, 84110, 84110, 3333, 77777, 1199000001, 5, E, WW, 08/06/2001, ONS Ent Name 1, ONS Ent Name 2, ONS Ent Name 3, ONS RU Name 1, ONS RU Name 2, ONS RU Name 3, , , , C, , 1, 102, S] due to field(s) [frosic92, rusic92]`